cute_tiled.h

```diff
@@ -1930,7 +1930,7 @@ int cute_tiled_read_properties_internal(cute_tiled_map_internal_t* m, cute_tiled
 			if (is_hex_color)
 			{
 				cute_tiled_expect(m, '"');
-				cute_tiled_read_hex_int(m, &prop.data.integer);
+				cute_tiled_read_hex_int(m, &prop.data.color);
```

Comment:
I am unclear if `.color` is used anywhere, but in my testing, I am getting the proper color properties with the correct alpha value passed through.